tests/README.md

```diff
@@ -0,0 +1,81 @@
+## Backend development
+
+The backend test suite can be run locally. 
+
+### Start the services
+
+We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
+
+    # ENSURE THE IMAGES ARE CLEAN
+    docker-compose down
+    # SETUP ALL IMAGES
+    docker-compose up --build
+
+### Install into virtual environment
+
+Treeherder, requires specific library versions that will likely interfere with with what you have installed. It is best to setup a virtual environment to contain the quirks it introduces.
+
+Be sure you are in the `treeherder` main directory
+
+    pip install virtualenv
+    virtualenv venv
+    source venv/bin/activate
+    venv/bin/pip install -r requirements/common.txt
+    venv/bin/pip install -r requirements/dev.txt
+    venv/bin/pip install -r requirements/docs.txt
+
+...or Windows...
+
+    pip install virtualenv
+    virtualenv venv
+    venv\Scripts\activate
+    python -m pip install -r requirements\common.txt
+    python -m pip install -r requirements\dev.txt
+    python -m pip install -r requirements\docs.txt  
+
+### Set environment variables
+
+Treeherder requires a number of environment variables that point to the services.  In our case, those services are in local docker containers.
+
+    ./tests/env.sh     # env.bat for Windows
+
+If you plan to use an IDE, here is the same, as a very long line
+
+```
+BROKER_URL=localhost//guest:guest@rabbitmq//;DATABASE_URL=mysql://root@localhost:3306/treeherder;REDIS_URL=redis://localhost:6379;SITE_URL=http://backend:8000/;TREEHERDER_DEBUG=True;TREEHERDER_DJANGO_SECRET_KEY=secret-key-of-at-least-50-characters-to-pass-check-deploy;NEW_RELIC_DEVELOPER_MODE=True
+```
+
+### Ensure everything is working
```

Comment:
This check is part of `runtests.sh` so I don't see the value of explicitely being called out.

Reply:
`runtests.sh` can not be run on windows

Reply:
Why not?

What can you run on Windows?

Reply:
@armenzg bash scripts do not run on windows. `python manage.py check` is a good command because it will inform the developer there are missing environment variables. `./runtests.sh` is a number of lint checks (which are not needed for project setup).  Running the full test suite at this time is premature; the setup is not-yet-complete.

Reply:
Interesting!

Running that check is really rare. It probably made a lot of sense to run it often as part of this PR but not necessary for others. If you feel strongly to keep it feel free to do so.

On another note, I've installed a Windows VM and you can run Bash scripts within Git Bash (part of Git for Windows). Also with VS Code you can switch the default interpreter from PowerShell to Git Bash. Let me know if you're interested to know more.